src/Api/Command/DebugAccesskeyCommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\ManagerBundle\Api\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+class DebugAccesskeyCommand extends Command
+{
+    /**
+     * @var string
+     */
+    private $projectDir;
+
+    /**
+     * @param string $projectDir
+     */
+    public function __construct(string $projectDir)
+    {
+        parent::__construct();
+
+        $this->projectDir = $projectDir;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this
+            ->setName('debug:access-key')
```

Comment:
The namespace is misleading. Every other command starting with `debug:` prints debug information so I would expect a `debug:access-key` to **print** my access key configuration and not to set or remove the access key.

Reply:
What about `access-key:set` and `access-key:remove`?

Reply:
Good point, changed in 14c921bdc99d960fa110c703b6db72a98d580dc6